SoftLayer/CLI/block/subnets/assign.py

```diff
@@ -0,0 +1,37 @@
+"""Assign block storage subnets to the given host id."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command()
+@click.argument('access_id', type=int)
+@click.option('--subnet-id', multiple=True, type=int,
+              help="ID of the subnets to assign; e.g.: --subnet-id 1234")
+@environment.pass_env
+def cli(env, access_id, subnet_id):
+    """Assign block storage subnets to the given host id.
+
+    access_id is the host_id obtained by: slcli block access-list <volume_id>
```

Comment:
Mention in the doc block of these function that these functions require https://sldn.softlayer.com/reference/datatypes/SoftLayer_Account/#iscsiisolationdisabled to be set to False